packages/navstar/buildinfo.json

```diff
@@ -4,7 +4,7 @@
     "navstar": {
       "kind": "git",
       "git": "https://github.com/dcos/navstar.git",
-      "ref": "27bbade72cfedb6a063860adfbbd5ba551ccafad",
+      "ref": "68997ac64fb6c6fbf1906323be565ace18756622",
       "ref_origin": "dcos/1.8"
```

Comment:
I don't see the equivalent of this commit anywhere on the `dcos/1.8` branch for `spartan`:
```
commit 927050af455802057d920d2adb03a84eefcd27e7
Merge: 553d0dc 0b5243b
Author: Deepak Goel <dgoel@mesosphere.com>
Date:   Thu Mar 23 01:03:24 2017 -0700

    Merge pull request #33 from dcos/fix-dcos-oss-804
    
    log message to include pid, module, function, line number
```

Am I missing something?